airgun/entities/host.py

```diff
@@ -84,6 +84,13 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def new_ui_button(self):
+        """Click New UI button and return the browser URL"""
```

Comment:
Suggestion: not to return anything from this function, just click on New UI button
```suggestion
        """Click New UI button"""
```